inc/osvr/Util/StdDeletable.h

```diff
@@ -22,25 +22,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef INCLUDED_StdDeletable_h_GUID_2C03AF21_E4B8_44B3_0F77_630666063711
-#define INCLUDED_StdDeletable_h_GUID_2C03AF21_E4B8_44B3_0F77_630666063711
+#ifndef INCLUDED_Deletable_h_GUID_66D6C45A_7AB7_4198_F8EE_202F0FCE682C
+#define INCLUDED_Deletable_h_GUID_66D6C45A_7AB7_4198_F8EE_202F0FCE682C
```

Comment:
I've no idea, is that auto-generated?

I think clang-format is what is used for keeping things tidy, but I've just been trying to carefully keep in sync with the surrounding code.